translations/ui-orders/en.json

```diff
@@ -760,5 +759,8 @@
   "requests.message.itemId": "Item ({itemId})",
   "requests.footer.close": "Close",
   "requests.request": "Request",
-  "requests.request.isOpened": "Yes"
+  "requests.request.isOpened": "Yes",
+
+  "poLine": "PO line",
+  "poLine.pluralized": "{count} {count, plural, one {PO line} other {PO lines}}"
```

Comment:
some brainfuck here...

Reply:
react-intl [doc](https://github.com/formatjs/react-intl)
the same in [e-holdings](https://github.com/folio-org/ui-eholdings/blob/e6adc9b5a757a99d4749ebb541bd98aa11e0e828/translations/ui-eholdings/en.json#L354) & [users](https://github.com/folio-org/ui-users/blob/964a701304344ba900235f8a52464e3f6c5d175e/translations/ui-users/en.json#L3) :)